README.md

```diff
@@ -1,2 +1,21 @@
 # open-source-funders
-Name and descrition TBA
+
+> Name and description TBA
+
+## Background
+
+Funding in open source is a continual problem. For a project to survive beyond the momentary interest of its maintainer, it needs funds. Today, there are more ways of funding open source projects and contributors than ever: GitHub Sponsors, Open Collective, npm.fund, Patreon, and so on. Thousands of projects have made it clear they need funds; hundreds of sponsors have helped out. But it's not always clear what companies do sponsor Open Source, and which companies don't. It's also not obvious which companies sponsor who on what platform. And while tens of thousands of companies have GitHub organization profiles, we suspect that only a few hundred companies actively sponsor the code that they use.
+
+This project highlights companies that have contributed to the open source ecosystem through funds directly. We showcase companies that have given money to open source projects through any open source platform, and we monitor the different ways they do so: either through direct payouts, sponsorships, paying for talented engineers to work full-time on open source, funding meetups or conferences, and so on.
```

Comment:
`This project highlights companies that have contributed to the open source ecosystem through funds directly` > `This project highlights companies that have funded the open source ecosystem`

Reply:
`paying for talented engineers to work full-time on open source` // is only full-time included? I’d think you meant here that they employ people to work on OSS, right?

Reply:
I think either works. I've edited this a bit.